File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height of the pyramid: ");
    } while (height <= 0 || height > 20);

    /*
    ...#  -> row 1, 1 hash, 3 spaces
    ..##  -> row 2, 2 hashes, 2 spaces
    .###  -> row 3, 3 hashes, 1 space
    ####  ->
    for each row -> spaces + hashes = total height - hashes
    */
    for (int row = 1; row <= height; row++)
    {
        for (int spaces = 1; spaces <= height - row; spaces++)
        {
            printf(" ");
        }
        //print hashtag for each row
        for (int hash = 1; hash <= row; hash++)
        {
            printf("#");
        }
        printf(" ");
        printf(" ");
        for (int hash = 1; hash <= row; hash++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: pset6/mariopy/mario.py
from cs50 import get_int
height = 0
while True:
    height = get_int("Height of the pyramid: ")
    if height > 0 and height < 9:
             break


for row in range(height):
    for spaces in range(height - row - 1):
        print(" ", end = "")
    for hash in range(row + 1):
        print("#", end = "")
    print(" ", end = "")
    print(" ", end = "")
    for hash in range(row + 1):
        print("#", end = "")
    print("")
